app/__init__.py

```diff
@@ -1,7 +1,35 @@
 from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+from dotenv import load_dotenv
+import os 
 
+db = SQLAlchemy()
+migrate = Migrate()
+load_dotenv()
 
 def create_app(test_config=None):
     app = Flask(__name__)
 
+    if not test_config:
+        app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+        app.config['SQLALCHEMY_DATABASE_URI'] = os.environ.get("SQLALCHEMY_DATABASE_URI")
+
+    else:
+        app.config["TESTING"] = True
+        app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+        app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get(
+            "SQLALCHEMY_TEST_DATABASE_URI")
+
+
+    from app.models.planet import Planet
+
+    db.init_app(app)
+    migrate.init_app(app, db)
+
+    from .routes.routes import planet_bp
```

Comment:
We may want to change the import to `planet_routes.py` since it contains the nested route for moons. 

And `planet_routes` is a much better file name to hold all the `planet` routes.

```suggestion
    from .routes.planet_routes import planet_bp
```